http.md

```diff
@@ -0,0 +1,32 @@
+# HTTP
+
+HTTP - short for Hypertext Transfer Protocol; The foundation of data communication for the World Wide Web.
+
+## Classification / acceptance questions
+
+### Apprentice aka _I know it exists_
+- I know what problem is solved by HTTP
+
+### Intermediate aka _I've been looking in the network tab_
+- I know what types of content can be transported via HTTP
+- I know the structure of a HTTP request/response
+- I can make practical use of the network tab to debug a problem
+- I know request types and response status codes
+
+### Advanced aha _?_
+- I'm familiar with security flaws that come with sending unencrypted data via HTTP and how we solve that problem.
```

Comment:
Are you referring to HTTPS? If yes, let's make it clear - without the keyword it's harder to google it.
If not, let's add a point about HTTPS